docs/learn-kusama-vs-polkadot.md

```diff
@@ -0,0 +1,57 @@
+---
+id: learn-kusama-vs-polkadot
+title: Comparing Polkadot and Kusama
+sidebar_label: Comparing Polkadot and Kusama
+---
+
+# What is the difference between Polkadot and Kusama?
```

Comment:
This will cause the page to have two header titles since the title is taken from the metadata above. So it would end up rendering like the below. I would recommend changing the `title` field in the metadata and deleting the first header.

# Comparing Polkadot and Kusama

# What is the difference between Polkadot and Kusama?

Although they are built...

Reply:
Changed